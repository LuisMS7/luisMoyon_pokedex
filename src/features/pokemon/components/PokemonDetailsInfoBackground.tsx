import React from "react";
import { StyleSheet, View } from "react-native";
import { perfectSize } from "@utils/perfect-size";

const PokemonDetailsInfoBackground = () => {
	return <View style={styles.container}></View>;
};

const styles = StyleSheet.create({
	container: {
		width: "95%",
		alignSelf: "center",
		backgroundColor: "white",
		borderTopRightRadius: perfectSize(18),
		borderTopLeftRadius: perfectSize(18),
		position: "absolute",
		height: "78%",
		bottom: 0,
		zIndex: -10,
		paddingTop: perfectSize(80),
		paddingHorizontal: perfectSize(20),
	},
});

export default PokemonDetailsInfoBackground;
