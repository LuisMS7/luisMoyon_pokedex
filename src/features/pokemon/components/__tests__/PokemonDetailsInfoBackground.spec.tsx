import React from "react";
import { render, screen } from "@testing-library/react-native";
// eslint-disable-next-line max-len
import PokemonDetailsInfoBackground from "@pokemon/components/PokemonDetailsInfoBackground";

describe("PokemonDetailsInfoBackground", () => {
	it("should render correctly", () => {
		render(<PokemonDetailsInfoBackground />);
		expect(screen.toJSON()).toMatchSnapshot();
	});
});
