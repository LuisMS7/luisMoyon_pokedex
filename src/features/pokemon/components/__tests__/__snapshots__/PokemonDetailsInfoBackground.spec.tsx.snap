// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PokemonDetailsInfoBackground should render correctly 1`] = `
<View
  style={
    {
      "alignSelf": "center",
      "backgroundColor": "white",
      "borderTopLeftRadius": 26.83792444178802,
      "borderTopRightRadius": 26.83792444178802,
      "bottom": 0,
      "height": "78%",
      "paddingHorizontal": 29.819916046431132,
      "paddingTop": 119.27966418572453,
      "position": "absolute",
      "width": "95%",
      "zIndex": -10,
    }
  }
/>
`;
